CHANGELOG.md

```diff
@@ -45,7 +45,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
   gtk-sharp
 - [SIL.Windows.Forms.SettingProtection] Deprecated ManageComponent method
 - [SIL.Scripture] VerseRef.TrySetVerseUnicode: Improve handling of non-decimal numerals and surrogate pair numerals (#1000)
-- [SIL.WritingSystems] Ignore deprecated region subtags (#763)
+- [SIL.WritingSystems] Ignore deprecated region subtags in `ScriptRegionVariantView`(#763)
```

Comment:
```suggestion
- [SIL.Windows.Forms.WritingSystems] Ignore deprecated region subtags in `ScriptRegionVariantView`(#763)
```

There should also be an entry under Added that mentions the additional `StandardSubtags.FilteredRegions` property.

And since we're adding a new property, the commit message should have a `+semver:minor` line.